aries_cloudagent/version.py

```diff
@@ -1,3 +1,3 @@
 """Library version information."""
 
-__version__ = "0.7.2"
+__version__ = "v0.7.3-rc0"
```

Comment:
No 'v' here 😲

Reply:
Fixed in docs and in code locations.  Doh.